### DISCUSSION TITLE
怎么确认应用的api接口地址呢 ？

### DISCUSSION THREAD

**Discussion by Techvh** [labring/FastGPT#521] [Q&A]
BaseURL 不是接口地址，而是所有接口的根地址，直接请求 BaseURL 是没有用的，那请问怎么确认应用的接口地址呢 ？

**Comment by c121914yu** [comment]
> BaseURL 不是接口地址，而是所有接口的根地址，直接请求 BaseURL 是没有用的，那请问怎么确认应用的接口地址呢 ？

可以看文档里的open api，每个接口都有例子
